src/Orleans/Orleans.csproj

```diff
@@ -37,6 +37,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjection)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" />
```

Comment:
please remove this one for now if it's not needed

Reply:
it is needed for LegacyLogger project, since it  have a dependency for Microsoft.Extensions.Logging and also Orleans. And Orleans need to have dependency on Microsoft.Extensions.Logging anyways, I'd like to make this change as soon as possible.

Reply:
As long as we don't have the, let's not pollute things here, as this is a high impact change